test/e2e/page-objects/pages/send/solana-send-page.ts

```diff
@@ -4,23 +4,28 @@ import { Driver } from '../../../webdriver/driver';
 class SendSolanaPage {
   private driver: Driver;
 
-  private readonly sendAmountInput = '#send-amount-input';
+  private readonly clearToAddressField = '#send-clear-button';
 
-  private readonly toAddressInput = '#send-to';
+  private readonly cancelButton = {
+    tag: 'button',
+    testId: 'send-cancel-button-snap-footer-button',
+  };
 
   private readonly continueButton = {
-    text: 'Continue',
     tag: 'button',
+    testId: 'send-submit-button-snap-footer-button',
   };
 
+  private readonly sendAmountInput = '#send-amount-input';
+
   private readonly swapCurrencyButton = '#send-swap-currency';
 
-  private readonly cancelButton = {
-    text: 'Cancel',
-    tag: 'button',
-  };
```

Comment:
this selector is incorrect. The button element doesn't have text, rather there is a nested span element inside. Now the selector has been updated after adding an id in the button